config/weaver.ini.example

```diff
@@ -41,7 +41,12 @@ cache.result.enabled = false
 #   responsibility to correctly map your server routes to internal routes after applying the server-side operation.
 
 # --- Weaver Configuration ---
-weaver.configuration = ems
+# See documentation for more details
+#   DEFAULT: current instance offers a minimal subset of operations as 'basic' OGC-API - Processes (i.e.: no deploy)
```

Comment:
Offering only built-in processes?

Reply:
Yes, that would be the idea, but I'm not even sure it is 100% valid. 
I have not extensively tested how it behaves in that case. 
Deploy requests are not blocked, so it is probably more like a partial ADES that doesn't handle specific cases.